contrib/epee/include/net/http_auth.h

```diff
@@ -45,15 +45,22 @@ namespace net_utils
     public:
       struct login
       {
-        login() = delete;
+        login() : username(), password() {}
+        login(std::string username_, std::string password_)
+          : username(std::move(username_)), password(std::move(password_))
```

Comment:
Out of curiosity, is this a better way in general than: login(const std::string &username_): username(username_) {} ?